### PR TITLE
Building block on statistical power

### PR DESCRIPTION
**Pull request by arutyunovv**
Hi @srosh2000 and @lachlandeer, 

I am opening a pull request for the building block on statistical power. It focuses on power analysis for experimental research designs and includes a brief example of how to conduct power analysis in R. 

Note that I added the files to the 'introduction' part of the 'causal inference' section, as there isn't really any other section which fits the topic of statistical power and I didn't want to create a separate section for just one building block. Let me know if you have any alternative suggestions as to the location for this building block. 

Please have a look when you can and let me know what improvements I could make. Thank you!

Best, 
Victor

### CODE REVIEW

**Comment by arutyunovv** [comment]
Hi @alexandervossen, 

Since you mentioned that you're available to review some PRs during the WOW meeting, could you please have a look at this one? The BB is about statistical power: what it is, why it's important, how it can be increased, and how power analysis can be used in experimental settings. It also includes a brief example in R on how to perform power analysis. 

Thanks!

Best, 
Victor

**Comment by alexandervossen** [comment]
Hi Victor! I had a look at it and it is very well written! I have two suggestions/questions, albeit I am not much of an experimenter myself-contrary to what my haircuts from the 90s would suggest :)

I feel the intro to the post is still a bit technical. Maybe try to use more layman terms. 
ChatGPT gave me the following when queried for a layman explanation: 

Sure! Statistical power is a concept in statistics that helps us understand how likely it is that a study or experiment will detect an effect when there is actually one to be found.
Imagine you're a detective trying to find clues at a crime scene. If you have a really bright flashlight (high statistical power), you’re more likely to spot the clues that are there. If your flashlight is dim (low statistical power), you might miss some important clues even though they’re right in front of you.

In the context of a study, the "clues" are the real effects or differences we want to detect, such as the effectiveness of a new drug compared to a placebo. The "flashlight" represents the design and size of the study:

1. **Sample Size**: Larger sample sizes usually provide higher power because they give a clearer picture of what's happening.
2. **Effect Size**: If the effect (or difference) you're looking for is big, it’s easier to detect, which increases power.
3. **Significance Level**: This is like setting a threshold for how sure you want to be before you say you've found something. A lower threshold (more strict criteria) makes it harder to detect effects, which can reduce power.
4. **Variability**: Less variability (or noise) in your data makes it easier to spot real effects, thus increasing power.

In summary, statistical power is the probability that your study will detect a real effect when it exists. A higher power means a better chance of detecting the truth, much like a brighter flashlight increases the chances of finding clues at a crime scene.

Maybe get some inspiration from this. 

Also I felt the exampüle at the end - altough not wrong- felt a bit weird. Which experimental study has treatments with more than 1000 participants? Maybe in Medicine but rarely in social science I would argue. Just some food for thought- well done!

A

**Comment by arutyunovv** [comment]
Many thanks for your comments @alexandervossen! 

I'll adjust the intro to simplify it; I definitely agree that's it's a little too technical. Regarding the example, I have come across some experimental papers in economics which have such large sample sizes ([Edelman, Luca, and Svirsky (2017)](https://www.aeaweb.org/articles?id=10.1257/app.20160213) is an interesting one), but of course it's not very common. I think it makes sense to provide a more relatable example, as you suggest.

I'll tag you and Lachlan once I've incorporated your feedback.

**Comment by alexandervossen** [comment]
can it now be merged @arutyunovv ?

**Comment by arutyunovv** [comment]
Hi @alexandervossen, yes - it can now be merged!

I've committed the changes you suggested and also revised some minor formatting things. Thanks again!
